ReferralPlease/ReferralPlease/ViewController.swift

```diff
@@ -167,7 +167,7 @@ extension ViewController: WKNavigationDelegate {
     func goToHome() {
         // login should go to home vc
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
-        guard let mainTabController = storyboard.instantiateViewController(withIdentifier: "mainTabViewController") as? MainTabController else
+        guard let mainTabController = storyboard.instantiateViewController(withIdentifier: "FirstTimeLoginFormViewController") as? MainTabController else
```

Comment:
If the user is not a first time user, don't we still go to the homepage instead of showing the first time login form?